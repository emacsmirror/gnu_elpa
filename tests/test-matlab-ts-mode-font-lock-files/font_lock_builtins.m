% -*- matlab-ts -*-
function varargout = font_lock_builtins(vararin)
    nargin
    disp(nargin)
    varargout = varargin
end
