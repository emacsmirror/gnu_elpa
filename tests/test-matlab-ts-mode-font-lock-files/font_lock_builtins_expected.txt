c ccc ccccccccc ccc
kkkkkkkk vvvvvvvvv o ffffffffffffffffffbvvvvvvvb
    BBBBBB
    BBBBbBBBBBBb
    vvvvvvvvv o BBBBBBBB
kkk

